test/dlc_tests/conftest.py

```diff
@@ -864,6 +864,28 @@ def skip_inductor_test(request):
             )
 
 
+@pytest.fixture(autouse=True)
+def skip_torchdata_test(request):
+    if "training" in request.fixturenames:
```

Comment:
These `if-else` conditions can be further optimized using a `for` loop. Something like:

```
lookup_fixtures = ["traininig", "pytorch_training", "inference", "pytorch_inference"]
image_uri = ""

for lookup_fixture in lookup_fixtures:
      if lookup_fixture in request.fixturenames:
            image_uri = request.getfixturevalue(lookup_fixture)
            break

if not image_uri:
    return
```